src/app/page.tsx

```diff
@@ -1,12 +1,27 @@
 import { Ranking } from "../components/ranking/ranking";
 import { mockedRanking, mockedTopScorers } from "../../mocks/data";
 import { TopScorers } from "@/components/top-scorers/top-scorers";
+import { getMatchDayByTimeframe } from "@/lib/actions/matchDays";
+import { MatchDay } from "@/components/bet/bet";
+import { getBets } from "@/lib/actions/bet";
+import { auth } from "@/lib/auth";
 
 const Home = async () => {
+  const session = await auth();
+  const userId = session?.user.id;
+  const currentMatchDay = await getMatchDayByTimeframe("current");
+  const bets = await getBets(currentMatchDay?._id, userId!);
+
   return (
-    <div>
-      <div className="flex w-full gap-20">
-        <div className="">
+    <div className="flex w-full flex-col gap-20">
+      {currentMatchDay ? (
+        <MatchDay
+          matchDayNumber={currentMatchDay.dayNumber}
+          bets={JSON.parse(JSON.stringify(bets))}
+        />
```

Comment:
Please add an empty state and I think it should be 50% width but this can be done when showing previous results

Reply:
https://github.com/miqs1992/eurotyper/blob/master/app/javascript/components/next_match_day.vue lines 6 -13